src/components/web-form-renderer.vue

```diff
@@ -40,6 +40,14 @@ except according to the terms contained in the LICENSE file.
             <a href="emailto:support@getodk.org">support@getodk.org</a>
           </template>
         </i18n-t>
+        <i18n-t v-else-if="submissionModal.type === 'sessionTimeoutModal'" tag="p" keypath="sessionTimeoutModal.body">
+          <template #here>
+              <a href="/login" target="_blank">{{ $t('sessionTimeoutModal.here') }}</a>
+          </template>
+          <template #supportEmail>
+            <a href="emailto:support@getodk.org">support@getodk.org</a>
+          </template>
```

Comment:
I don't see `{supportEmail}` mentioned in the `sessionTimeoutModal.body` message.